updatecli/weekly.d/jenkinscontroller-agents-packer-agent-all-in-one.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name: Bump Packer Agent Templates (all-in-one) Version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  packerImageVersion:
+    kind: githubrelease
+    spec:
+      owner: "jenkins-infra"
+      repository: "packer-images"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+
+sources:
+  getLatestInboundAllInOneContainerImageX86:
+    kind: dockerdigest
+    name: Get digest of the jenkinsciinfra/jenkins-agent-ubuntu-22.04 image
+    dependson:
+      - packerImageVersion
+    spec:
+      image: "jenkinsciinfra/jenkins-agent-ubuntu-22.04"
+      tag: '{{ source "packerImageVersion"}}'
+      architecture: amd64
+
+conditions:
+  checkAllInOneContainerImages:
+    disablesourceinput: true
+    name: Check that x86 and arm64 all-in-one images are different
+    kind: shell
+    spec:
+      command: test {{ source "getLatestInboundAllInOneContainerImageX86" }} != {{ source "getLatestInboundAllInOneContainerImageARM" }}

```

Comment:
This line triggers the following error:

```
Pipeline "Bump Packer Agent Templates (all-in-one) Version" failed
Skipping due to:
        sources stage:  "template: cfg:45:89: executing \"cfg\" at <source \"getLatestInboundAllInOneContainerImageARM\">: error calling source: key not defined in configuration"
```

tip: there is something missing ;)

Reply:
Thank you for the tip; it was extremely helpful in resolving this issue.